src/arrays.js

```diff
@@ -1,62 +1,73 @@
 const getNthElement = (index, array) => {
-  // your code here
-};
+  const newIndex = 0;
```

Comment:
This is a little over engineered here, could be refactored by using the modulo operator :)
````
return array[index % array.length];
`````